flyteadmin/pkg/rpc/adminservice/project.go

```diff
@@ -28,6 +28,20 @@ func (m *AdminService) RegisterProject(ctx context.Context, request *admin.Proje
 	return response, nil
 }
 
+func (m *AdminService) GetDomains(ctx context.Context, request *admin.GetDomainRequest) (*admin.Domains, error) {
+	defer m.interceptPanic(ctx, request)
+	if request == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Incorrect request, nil requests not allowed")
+	}
+	var response *admin.Domains
+	m.Metrics.projectEndpointMetrics.get.Time(func() {
```

Comment:
you can add another metric for get domain

Reply:
got it!